### ISSUE TITLE
Playnite not loading extension

### ISSUE THREAD

**Issue by Linx-ESP** [hawkeye116477/playnite-legendary-plugin#15]
This from the logs could be related:

```
30-05 00:19:25.781|ERROR|ExtensionFactory:Failed to load plugin: Legendary (Epic Games) library integration
System.ArgumentException: Caracteres no válidos en la ruta de acceso.
   en System.IO.LongPathHelper.Normalize(String path, UInt32 maxPathLength, Boolean checkInvalidCharacters, Boolean expandShortPaths)
   en System.IO.Path.NewNormalizePath(String path, Int32 maxPathLength, Boolean expandShortPaths)
   en System.IO.Path.NormalizePath(String path, Boolean fullCheck, Int32 maxPathLength, Boolean expandShortPaths)
   en System.IO.Path.GetFullPathInternal(String path)
   en LegendaryLibraryNS.LegendaryLauncher.get_DefaultGamesInstallationPath() en C:\Users\Bartek\Documents\git\playnite-legendary-plugin\src\LegendaryLauncher.cs:línea 85
   en LegendaryLibraryNS.LegendaryLibrarySettings..ctor() en C:\Users\Bartek\Documents\git\playnite-legendary-plugin\src\LegendaryLibrarySettingsViewModel.cs:línea 26
   en LegendaryLibraryNS.LegendaryLibrarySettingsViewModel..ctor(LegendaryLibrary library, IPlayniteAPI api) en C:\Users\Bartek\Documents\git\playnite-legendary-plugin\src\LegendaryLibrarySettingsViewModel.cs:línea 55
   en LegendaryLibraryNS.LegendaryLibrary..ctor(IPlayniteAPI api) en C:\Users\Bartek\Documents\git\playnite-legendary-plugin\src\LegendaryLibrary.cs:línea 41
```

Should it be ``C:\Users\ (My User) \Documents\...`` ? Bartek is not my windows user   
   
Tried with both, 1.0 and 1.1.0, downloaded from inside playnite and github.

**Comment by hawkeye116477** [comment]
What is your Playnite's version?

**Comment by Linx-ESP** [comment]
Windows 11 - 22621.1778
Playnite 10.16
%userprofile% is "C:\Users\JohnS"

Playnite is installad on its default location, appdata/local/playnite
I tried with legendary on Program Files and outside in a non-admin path, sure it didn't have special characters nor long name.

**Comment by hawkeye116477** [comment]
Hmm, so it's really weird, should be fine. You can try tips from https://www.epicgames.com/help/en-US/epic-games-store-c5719341124379/error-codes-c5719359691291/the-folder-path-contains-invalid-character-error-a6764659110683, maybe that will help.

**Comment by Linx-ESP** [comment]
Solved with 1.1.1
